Makefile

```diff
@@ -47,9 +49,10 @@ generate-$(1):
 generate-tests: generate-$(1)
 endef
 
-RUNNERS := $(wildcard $(RUNNERS_DIR)/*.py)
-RUNNERS := $(RUNNERS:$(RUNNERS_DIR)/%=%)
-RUNNERS := $(basename $(RUNNERS))
+RUNNERS_FOUND := $(wildcard $(RUNNERS_DIR)/*.py)
+RUNNERS_FOUND := $(RUNNERS_FOUND:$(RUNNERS_DIR)/%=%)
+RUNNERS_FOUND := $(basename $(RUNNERS_FOUND))
```

Comment:
Can we use this variable when calling the `runner_gen` macro? Otherwise we get lots of "empty runs" I mentioned in my previous comment.

Reply:
Maybe the variable names are misleading but `RUNNERS` is the variable containing runners that passed the checks and `RUNNERS_FOUND` contains all runners which are present in `tools/runners`.

Reply:
Ok, great, thanks!